pkgs/development/python-modules/pyregion/default.nix

```diff
@@ -47,6 +47,11 @@ buildPythonPackage rec {
 
   nativeCheckInputs = [ pytestCheckHook pytest-astropy ];
 
+  disabledTests = [
```

Comment:
Perhaps an update to the latest release would help to get all tests to pass. The release included https://github.com/astropy/pyregion/pull/161 which seems to touch some tests.

Reply:
Errors with:

> pyregion/__init__.py:15: in <module>
>     from .version import __version__
> E   ModuleNotFoundError: No module named 'pyregion.version'

https://github.com/astropy/pyregion/blob/main/pyregion/__init__.py

Any clue?

Reply:
I don't know why the build of 2.2.0 is failing. Let's fix the test phase  in 2.1.1 for the moment, we will deal with the upgrade later.

Reply:
Are tests failing on darwin too?

Reply:
Not sure. I takes forever to build this branch.

Reply:
> Not sure. I takes forever to build this branch.

I have switched branch to master so you can test.

Reply:
Never the less please add a comment why the tests are disabled.

Reply:
```nix
  disabledTests = lib.optional (!stdenv.hostPlatform.isDarwin)
    [
    "test_calculate_rotation_angle"
    "test_region"
    ];
```

Reply:
@SuperSandro2000 I have added an explanation and full logs.

I don't want to judge on why it is failing. Because I don't have domain knowledge. I only wanted to mark package broken to signal it is breaking before my build even started and is unrelated.

Because I know people are going to ask for a fix or to skip tests, I went for skipping tests instead. But that is not really what is driving me to do this. 

I feel the Python pain everyday despite not using Python. Because it's endless breakage leaks all the time.